### PR TITLE
CyberProof connector

### PR DESCRIPTION
**Pull request by DavidMeu**
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

- [x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [ ] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [ ] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [ ] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

### CODE REVIEW

**Comment by DavidMeu** [comment]
Attaching custom connector testing:
image

**Comment by natalie-pienkowska** [comment]
Hi @DavidMeu! Our certification engineer, Srikanth, is currently on emergency leave. Please expect a delay with your PR. He will return January 31st, 2022. We apologize for any inconveniences.

**Comment by DavidMeu** [comment]
@natalie-pienkowska
Thank you for announcing. 
Any progress?

File: certified-connectors/CyberProof/apiDefinition.swagger.json [hunk 1]
```diff
@@ -0,0 +1,225 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.0",
+    "title": "CyberProof Actions",
+    "description": "This connector will enable you to execute CyberProof Actions.",
```
Comment (sriyen-msft) [inline certified-connectors/CyberProof/apiDefinition.swagger.json:6]: @DavidMeu Your connector description is short and vague that everyone can not understand about your product or service. Please follow the instructions mentioned in the [documentation](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#write-a-description-for-your-connector) and have a detailed description for your connector that should describe concisely the main purpose and value offered by your connector.

File: certified-connectors/CyberProof/readme.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# CyberProof
```
Comment (sriyen-msft) [inline certified-connectors/CyberProof/readme.md:1]: @DavidMeu Please follow the [templates](https://github.com/microsoft/PowerPlatformConnectors/tree/dev/templates/certified-connectors) we have provided, and have the required information in your connector documentation files.

File: certified-connectors/CyberProof/apiProperties.json [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+        "parameters": {
+          "x-ms-apimTemplateParameter.name": "St2-Api-Key",
+          "x-ms-apimTemplateParameter.value": "@connectionParameters('api_key')",
+          "x-ms-apimTemplateParameter.existsAction": "override",
+          "x-ms-apimTemplate-policySection": "Request"
+        }
+      }
+    ],
+    "publisher": "CyberProof Inc.",
+    "stackOwner": "CyberProof Inc."
```
Comment (sriyen-msft) [inline certified-connectors/CyberProof/apiProperties.json:50]: @DavidMeu You have provided stack owner as "CyberProof Inc." and submitted privacy policy from UST.com https://www.ust.com/en/legal-notice/privacy-policy.

Can you please let me know if "CyberProof Inc." is a fully qualified organization to publish this connector.

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

File: certified-connectors/CyberProof/apiProperties.json (hunk 3 above)
Comment (DavidMeu) [reply certified-connectors/CyberProof/apiProperties.json:50]: Yes it is.
"CyberProof Inc." is a subsidiary of UST

File: certified-connectors/CyberProof/apiDefinition.swagger.json (hunk 1 above)
Comment (DavidMeu) [reply certified-connectors/CyberProof/apiDefinition.swagger.json]: @sriyen-msft 
Updated

File: certified-connectors/CyberProof/readme.md (hunk 2 above)
Comment (DavidMeu) [reply certified-connectors/CyberProof/readme.md]: @sriyen-msft 
Followed

**Comment by DavidMeu** [comment]
@sriyen-msft 
Thank you for the reviewing.
Looking forward for our connector to get approved.
e.g
Added company icon (not sure it is needed here if not will appriciate your guidance)

**Comment by Amjed-Ayoub** [comment]
Dear partner,

We apologize for the delay you might be experiencing while we are working on a high volume of connector requests. Rest assured that we are doing everything we can to address this volume. We are taking measures like improving our processes and hiring additional team members to guarantee resolving your connector requests in a timely manner. 

Please let us know if you have any questions in the meantime.

**Comment by sriyen-msft** [comment]
@DavidMeu Please remove the icon from the PR which is not required to open source. The icon is something is required to submit in the ISV studio to get your connector certified.

**Comment by DavidMeu** [comment]
> @DavidMeu Please remove the icon from the PR which is not required to open source. The icon is something is required to submit in the ISV studio to get your connector certified.

Removed

**Comment by DavidMeu** [comment]
@Amjed-Ayoub @sriyen-msft 
Can we proceed with merging this connector?

**Review by sriyen-msft — APPROVED** [review]

**Pull request merged**